plugins/gatsby-theme-iterative-docs/src/utils/front/useCustomYtEmbeds.ts

```diff
@@ -24,7 +25,7 @@ const setUpEmbedClickListeners = (embeds: NodeListOf<Element>) => {
         iframe.src = iframe?.src + `&autoplay=1`
       }
       hideAllEmbedOverlays(embeds)
-      localStorage.setItem('yt-embed-consent', 'true')
+      safeLocalStorage.setItem('yt-embed-consent', 'true')
```

Comment:
or maybe use optional chaining(`?.`) instead?

Reply:
As mentioned in the previous comment, the issue isn't that `localStorage` doesn't exist, it's about that using it will cause a `SecurityError` :)